contrib/golang/filters/http/source/golang_filter.cc

```diff
@@ -917,6 +917,37 @@ CAPIStatus Filter::clearRouteCache() {
   return CAPIStatus::CAPIOK;
 }
 
+CAPIStatus Filter::refreshRouteCache() {
```

Comment:
Maybe it's better to add a bool param in `clearRouteCache(bool refresh)`? it may save some similar code.
Adding a new `RefreshRouteCache` Golang method is friendly for users.

Reply:
@doujiang24 
Updated.